### PR TITLE
Move nuget from version 6.4.2 to 6.4.3

### PR DESCRIPTION
**Pull request by chcosta**
Validated with https://dev.azure.com/dnceng/internal/_build/results?buildId=2385899&view=results

Fixes https://github.com/dotnet/dnceng/issues/2022

### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/arcade/tree/main/Documentation/Validation

### CODE REVIEW

**Review by garath — APPROVED** [review]

**Comment by chcosta** [comment]
Test failure needs investigation.  Closing this PR and moving the issue to the backlog, it will need to be picked up by someone else.

File: global.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {
+  "sdk": {
+    "version": "7.0.116",
+    "rollForward": "disable"
```
Comment (dougbu) [inline global.json:4]: nit, to be more consistent w/ at least release/6.0 (where we've avoided this issue):
```suggestion
    "rollForward": "latestFeature"
```

Comment (dougbu) [reply global.json:4]: I also suggest porting this update to release/8.0 and main. we'll be scratching our heads again too soon otherwise

**Comment by chcosta** [comment]
🚢

**Pull request merged**